mythril/analysis/modules/external_calls.py

```diff
@@ -5,6 +5,7 @@
 from mythril.analysis.swc_data import REENTRANCY
 from mythril.analysis.modules.base import DetectionModule
 from mythril.analysis.report import Issue
+from mythril.analysis.call_helpers import get_call_from_state
```

Comment:
could you just revert this module to the previous version? The changes added to this module only add a dependency to the helpers, without adding any features.

Call should never be None, as the function is only called for call instructions.

Reply:
I think Call returns None for internal calls. maybe i should change the name of that function later.